log/logger_test.go

```diff
@@ -124,7 +124,7 @@ func Test_Caller(t *testing.T) {
 	a, buffer, log := Setup(t)
 
 	log.Info().With(StackTrace).Logf("message")
-	a.Regexp(regexp.MustCompile(`caller_0=(.*?)(\/log\/logger_test\.go)`), buffer.String())
+	a.Regexp(regexp.MustCompile(`caller_\d=(.*?)(\/log\/logger_test\.go)`), buffer.String())
```

Comment:
We also want to make sure its `called_0` so the first element of the stack where log was called.